### PR TITLE
allow all too-hot params to be configured

### PR DESCRIPTION
**Pull request by staltz**
This PR is better than #191

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -370,6 +370,18 @@ const config = {
      * Default: Infinity
      */
     maxCpu: 85,
+
+    /** This applies only if `maxCpu` is defined.
+     * See `maxPause` in the module `too-hot`, for its definition.
+     * Default: 300
```
Comment (arj03) [inline README.md:376]: 180

Comment (arj03) [reply README.md:376]: or is it the default of too-hot?

Comment (staltz) [reply README.md:376]: Check line 12 in log.js, I kept it at 300. That's because 300 is good in CI. 180 is good for mobile.

Comment (arj03) [reply README.md:376]: Ah right, this is from the README :)

**Review by arj03 — APPROVED** [review]

**Pull request merged**